superset-frontend/images/superset-logo-horiz.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Where is the SVG used? I don't see it being used anywhere

Reply:
It's basically vestigial. The "thing I wanted to try" was editing this SVG file to use "currentColor" for the blue fill areas, such that the SVG inherits the brandColor variable. Then if people wanted to restyle their superset instance to use orange (or whatever) in the brand color, it'd be reflected in the logo.

However, that basically requires building in some JSX that depends on the logo using SVG. I'm sure lots of people are using JPG/PNG logos, so a big revamp around this suddenly out of scope.

TL;DR: it's a bit vestigial for now, but I hope to circle back and make good use of it. I can remove it from the PR if it's a bother.